src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -4710,8 +4711,10 @@ void TLayout::layoutStaffTypeChange(const StaffTypeChange* item, StaffTypeChange
     }
 }
 
-static void layoutStem(const Stem* item, const LayoutContext& ctx, Stem::LayoutData* ldata)
+void TLayout::layoutStem(const Stem* item, Stem::LayoutData* ldata, const LayoutConfiguration& conf)
 {
+    LD_INDEPENDENT;
```

Comment:
It looks like `Note::mirror` (as opposed to `Note::userMirror`) should be layout data, and Stem depends on it

Reply:
Actually the same goes for Stem::length, which is calculated outside `layoutStem`

Reply:
> It looks like Note::mirror (as opposed to Note::userMirror) should be layout data, and Stem depends on it

fixed 

> Actually the same goes for Stem::length, which is calculated outside layoutStem

I would like to deal with this separately